packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/translations/Messages_fr_FR.json

```diff
@@ -21,6 +21,7 @@
   "order_catalog_price_interval_month_by_ip": "mois/IP",
   "ip_agora_ipv6_description": "Lorsque de grands blocs IP sont nécessaires et qu'un schéma d'adressage hiérarchique est défini, l'IPv6 est le meilleur choix. Assurez-vous que tous vos clients sont en mesure d'utiliser le nouveau protocole ainsi que vos services",
   "free_price": "gratuit",
+  "ip_agora_ipv6_beta": "beta",
```

Comment:
Should "beta" be translated ?

Reply:
Its should be constant as per existing constant in https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/components/project/storages/databases/engines.constants.js#L11

Reply:
I think should translate bc in some translation files PL language is "Wersja Beta"